용재/1935.py

```diff
@@ -0,0 +1,27 @@
+N = int(input())
+getText = input()
+
+inputNumbers = []
+
+for i in range(N):
+  inputNumbers.append(int(input()))
+
+stack = []
+
+for each in getText:
+  if 'A' <= each <= 'Z':
+    stack.append(inputNumbers[ord(each) - 65])
+  else:
+    right = stack.pop()
+    left = stack.pop()
+    if each == '+':
+      stack.append(left + right)
+    if each == '-':
+      stack.append(left - right)
+    if each == '*':
+      stack.append(left * right)
+    if each == '/':
+      stack.append(left / right)
```

Comment:
if문만을 사용하여 이미 `+`연산을 수행하였지만 `each` 변수가
`*`, `-`, `/`이 세가지 연산인지 아닌지를 추가적으로 체크하는 경우가 발생할 수 있을 것 같습니다!

입력으로 잘못된 후위표기식이 들어오는 경우가 없기 때문에
`elif`문을 사용하여 다음과 같이 작성할 수 있을 것 같습니다!

```suggestion
    if each == '+':
      stack.append(left + right)
    elif each == '-':
      stack.append(left - right)
    elif each == '*':
      stack.append(left * right)
    else:
      stack.append(left / right)
```

Reply:
아하 예외처리가 필요한 상황이 있을 수 있겠네요. 그러면 확실히 elif를 써야 할 것 같습니다!